src/lib/logger/src/bindings.rs

```diff
@@ -1,4 +1,9 @@
-/* automatically generated by rust-bindgen */
+/* automatically generated by rust-bindgen 0.60.1 */
```

Comment:
Optional: maybe avoid printing the bindgen/cbindgen version numbers in the bindings, otherwise I don't think dependabot will ever be able to upgrade the bindgen version and we'll always have to do it manually.